### PR TITLE
[xml] Update Corsican translation for Notepad++ 8.7.2

### PR DESCRIPTION
**Pull request by Patriccollu**
Hello,

This is an update of **Corsican** localization to take into account the following commits:

 * https://github.com/notepad-plus-plus/notepad-plus-plus/commit/ba3b4578b2e29c86a4fabb5b6b6dd75a44fcdb60 Notepad++ 8.7.1 release
 * https://github.com/notepad-plus-plus/notepad-plus-plus/commit/ce58b424bdeec3616943f9ea8cd257f66700c3c0 Add Pin tab feature
 * https://github.com/notepad-plus-plus/notepad-plus-plus/commit/b8224808bdb6f9eaf94c066219634715703828da Add ability to open/copy selected files from Search-results
 * https://github.com/notepad-plus-plus/notepad-plus-plus/commit/eeb47530449cebabc9c94a8bccefc0f574a2491c Add "Minimize / Close to" option for System tray

Updated on November 20<sup>th</sup>:
 * https://github.com/notepad-plus-plus/notepad-plus-plus/commit/467182602a4331342015f5ad99a295e76b9da8bd Fix selected pathnames in search results localization issue
 * https://github.com/notepad-plus-plus/notepad-plus-plus/commit/c021c861959765db9efa0a7d6b156538a0e6f4db Code enhancement for file checking

Cheers,
Patriccollu.

### CODE REVIEW

File: PowerEditor/installer/nativeLang/corsican.xml [hunk 1]
```diff
@@ -1747,9 +1749,9 @@ Circà in tutti i schedarii ma esclude tutti i cartulari è sottucartulari log o
 			<finder-collapse-all value="Tuttu ripiegà"/>
 			<finder-uncollapse-all value="Tuttu spiegà"/>
 			<finder-copy value="Cupià a(e) linea(e) selezziunata(e)"/>
-			<finder-copy-paths value="Cupià u(i) nome(i) di chjassu"/>
+			<finder-copy-paths value="Cupià i nomi di chjassu selezziunati"/>
 			<finder-clear-all value="Tuttu viutà"/>
-			<finder-open-all value="Tuttu apre"/>
+			<finder-open-all value="Apre i nomi di chjassu selezziunati"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/corsican.xml:1754]: Need to change

**Review by donho — CHANGES_REQUESTED** [review]
Please update the localization with the one on master

File: PowerEditor/installer/nativeLang/corsican.xml [hunk 2]
```diff
@@ -1747,9 +1749,9 @@ Circà in tutti i schedarii ma esclude tutti i cartulari è sottucartulari log o
 			<finder-collapse-all value="Tuttu ripiegà"/>
 			<finder-uncollapse-all value="Tuttu spiegà"/>
 			<finder-copy value="Cupià a(e) linea(e) selezziunata(e)"/>
-			<finder-copy-paths value="Cupià u(i) nome(i) di chjassu"/>
+			<finder-copy-paths value="Cupià i nomi di chjassu selezziunati"/>
```
[the remark this answers is not in the document]
Comment (Patriccollu) [reply PowerEditor/installer/nativeLang/corsican.xml]: Code changed as requested.

File: PowerEditor/installer/nativeLang/corsican.xml (hunk 1 above)
Comment (Patriccollu) [reply PowerEditor/installer/nativeLang/corsican.xml]: Code changed as requested.

**Comment by Patriccollu** [comment]
@donho
I changed the code as requested and added translated string for latest commit about 'Code enhancement for file checking'.

**Review by donho — APPROVED** [review]

**Pull request closed without merging**